tensorflow/c/c_api_experimental.cc

```diff
@@ -32,8 +32,12 @@ limitations under the License.
 #include "tensorflow/core/distributed_runtime/rpc/grpc_server_lib.h"
 #include "tensorflow/core/framework/collective.h"
 #include "tensorflow/core/framework/node_def.pb.h"
+#include "tensorflow/core/framework/resource_var.h"
 #include "tensorflow/core/framework/shape_inference.h"
 #include "tensorflow/core/framework/tensor.pb.h"
+#include "tensorflow/core/framework/register_types.h"
+#include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/platform/types.h"
```

Comment:
I believe we only need these two files to solve the compilation problem (to make the include / dependency scope smaller):
```suggestion
#include "tensorflow/core/framework/resource_mgr.h"
#include "tensorflow/core/framework/variant.h"
```

Reply:
Good point, done.